extensions/vertx-graphql/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: "Eclipse Vert.x GraphQL"
+metadata:
+  keywords:
+  - "eclipse"
+  - "vertx"
+  - "vert.x"
+  - "graphql"
+  - "vertx-graphql"
+  categories:
+    - "web"
+  status: "stable"
```

Comment:
Not sure this extension is considered stable already?

Reply:
I think this one can be considered stable. It's not the SmallRye GraphQL one, it's a low level library.